scripts/check

```diff
@@ -6,7 +6,7 @@ if [ -d '.venv' ] ; then
 fi
 export SOURCE_FILES="hishel tests"
 
-${PREFIX}ruff format $SOURCE_FILES --diff
-${PREFIX}ruff check $SOURCE_FILES
+${PREFIX}ruff format $SOURCE_FILES
+${PREFIX}ruff check --fix $SOURCE_FILES
```

Comment:
I think, you need to run ./scripts/lint for fixing

Reply:
Yes, this was intended not to be commited, thanks for the lint-hint.